build.gradle

```diff
@@ -21,137 +21,185 @@ plugins {
+                "org.reflections:reflections",
+                "org.apache.commons:commons-lang3",
+        ],
+        sun_jaxb              : [
+                "com.sun.xml.bind:jaxb-impl",
+                "com.sun.xml.bind:jaxb-core",
+        ],
+        core_testing          : [
+                "org.junit.jupiter:junit-jupiter-api",
+                "org.junit.jupiter:junit-jupiter-params",
+                "org.junit.jupiter:junit-jupiter-engine",
+                "org.junit.vintage:junit-vintage-engine",
+                "org.hamcrest:hamcrest-library",
+                "org.hamcrest:hamcrest-core",
+                "org.mockito:mockito-junit-jupiter",
+                "org.mockito:mockito-core",
+                "org.objenesis:objenesis",
+                "net.bytebuddy:byte-buddy",
+                "net.bytebuddy:byte-buddy-agent",
+                "junit:junit",
+        ],
+        logging_deps          : [
+                "ch.qos.logback:logback-core",
+                "ch.qos.logback:logback-classic",
+                // Artifacts that bridge other logging frameworks to slf4j. Mime4j uses
+                // JCL for example.
+                "org.slf4j:jcl-over-slf4j",
+                "org.slf4j:log4j-over-slf4j",
+                "org.slf4j:slf4j-api",
+                "net.logstash.logback:logstash-logback-encoder",
+        ],
+        javax                 : [
+                "org.hibernate.javax.persistence:hibernate-jpa-2.1-api",
+                "javax.transaction:jta",
+                "javax.persistence:javax.persistence-api",
+                "javax.xml.bind:jaxb-api",
+        ],
+        commons               : [
+                "commons-codec:commons-codec",
+                "commons-collections:commons-collections",
+                "commons-io:commons-io",
+                "commons-lang:commons-lang",
+        ],
+        guice                 : [
+                "com.google.inject.extensions:guice-assistedinject",
+                "com.google.inject.extensions:guice-multibindings",
+                "com.google.inject.extensions:guice-servlet",
+                "com.google.inject.extensions:guice-throwingproviders",
+                "com.google.inject.extensions:guice-persist",
+                "com.google.inject:guice",
+                "aopalliance:aopalliance",
+                "javax.inject:javax.inject",
+        ],
+        liquibase             : "org.liquibase:liquibase-core",
+        liquibase_slf4j       : "com.mattbertolini:liquibase-slf4j",
+        oauth                 : [
+                "net.oauth.core:oauth",
+                "net.oauth.core:oauth-provider",
+        ],
+        collections           : "com.google.guava:guava:25.1-jre",
+        hibernate_validator_ap: "org.hibernate.validator:hibernate-validator-annotation-processor",
+        checkstyle            : [
+                "com.puppycrawl.tools:checkstyle",
+                "com.github.sevntu-checkstyle:sevntu-checks"
```

Comment:
Please remove, pulled in transitively by checkstyle.

Reply:
After removing "com.github.sevntu-checkstyle:sevntu-checks",  checkstyleMain and checkstyleTest tasks are failing.